pkg/atmos/component-helper/suite.go

```diff
@@ -72,16 +72,16 @@ func (s *TestSuite) getMergedVars(t *testing.T, additionalVars *map[string]inter
 func (s *TestSuite) DeployAtmosComponent(t *testing.T, componentName string, stackName string, additionalVars *map[string]interface{}) (*atmos.Options, string) {
 	phaseName := fmt.Sprintf("deploy/atmos component/%s/%s", stackName, componentName)
 
-	if s.Config.SkipDeployComponent {
-		s.logPhaseStatus(phaseName, "skipped")
-		return nil, ""
-	}
-
 	s.logPhaseStatus(phaseName, "started")
 
 	mergedVars := s.getMergedVars(t, additionalVars)
 	atmosOptions := getAtmosOptions(t, s.Config, componentName, stackName, &mergedVars)
 
+	if s.Config.SkipDeployComponent {
+		s.logPhaseStatus(phaseName, "skipped")
+		return atmosOptions, ""
+	}
+
```

Comment:
This change isn't mentioned in the PR description. Please mention it or move this change to another PR, as appropriate.

Reply:
Added to PR description